test_ptrack/test_views.py

```diff
@@ -2,7 +2,10 @@
 import re
 from mock import MagicMock
 from tempfile import NamedTemporaryFile
-from django.core.urlresolvers import reverse
+try:
+    from django.urls import reverse
+except ImportError:
+    from django.core.urlresolvers import reverse
```

Comment:
If we don't care about Django versions under 1.10 we could remove this line. But for now I kept it.